core/sql/executor/ex_hash_grby.h

```diff
@@ -302,6 +312,7 @@ NA_EIDPROC
   void resetClusterAndReadFromChild(); // Tmobile.
 
 public:
+  static const char *HashGrbyPhaseStr[];
```

Comment:
That's correct. Static global variables are also thread safe and I am hoping these phases are not changed forever and hence it shouldn't be error prone.